exp/api/v1alpha3/types.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
+
+import (
+	infrav1 "sigs.k8s.io/cluster-api-provider-azure/api/v1alpha3"
+)
+
+type (
+	VMSSVM struct {
+		ID               string          `json:"id,omitempty"`
```

Comment:
A VMSS instance has an ID and and instance ID? :P

Reply:
Indeed. `ID` is the ARM resource path (URI to resource) where `InstanceID` is the VMSS scoped instance ID ("a" or "0").

I'm inclined to think you are joking around, but just wanted to leave a note on the difference. [Docs here](https://docs.microsoft.com/en-us/rest/api/compute/virtualmachinescalesetvms/list#virtualmachinescalesetvm).